llvm/lib/Object/ELF.cpp

```diff
@@ -191,6 +191,18 @@ StringRef llvm::object::getELFRelocationTypeName(uint32_t Machine,
 
 #undef ELF_RELOC
 
+#define ELF_RISCV_NONSTANDARD_RELOC(vendor, name, number)                      \
+  if (Vendor == #vendor && Type == number)                                     \
+    return #name;
```

Comment:
Can you put this inside the function body? Same for the `#undef`

Reply:
I was copying the style from `getELFRelocationTypeName`, but sure.